sklearn/feature_selection/base.py

```diff
@@ -120,3 +120,72 @@ def inverse_transform(self, X):
         Xt = np.zeros((X.shape[0], support.size), dtype=X.dtype)
         Xt[:, support] = X
         return Xt
+
+
+def featurewise_scorer(score_func, absolute_score=True, **kwargs):
+    """ A wrapper function around score functions.
+
+    Parameters
+    ----------
+    score_func : callable
+        Function taking arrays X and y, and returning a pair of arrays
```

Comment:
Doesn't it just return a pair of numbers?

Reply:
I will change the entire description appropriately.